src/actions/matches.js

```diff
@@ -2,11 +2,18 @@ import { database } from './../store';
 import matchModel from './../models/matches';
 
 // set match
-export const addMatch = (userA, userB) => {
+export const addMatch = (users) => {
     let key = database.ref('/matches').push().key;
     let model = matchModel({
-        userA: userA.uid,
-        userB: userB.uid
+        competitors: users
     });
-    return database.ref('/matches/' + key).set(model);
+    return database.ref(`/matches/${key}`).set(model);
+}
+
+export const declareWinner = (matchId, winner, declarer) => {
+    return database.ref(`/matches/${matchId}/winners/${declarer.uid}`).set(winner.uid);
+}
+
+export const removeMatch = (matchId) => {
+    return database.ref(`/matches/${matchId}`).remove();
```

Comment:
Do we want to remove the match or just change the state of the match like `inProgress=false`?
If you preserve the matching history we can use this to show previous matches per player.

Reply:
At the moment you can only remove a match, when the match is not completed.
so only when not all the competitors have voted for the same winner.
But yeah maybe I should put this check in the remove function also, so you can never use this function any other way